File: 07. Perulangan Lanjut/C. Pola III/c_pola_iii.java
import java.util.Scanner;

class c_pola_iii {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        scan.close();
        int counter = 0;
        for (int i = 0; i < N; i++) {
            for (int j = 0; j < i + 1; j++) {
                System.out.print(counter % 10);
                counter++;
            }
            System.out.println();
        }
    }
}

File: 07. Perulangan Lanjut/C. Pola III/c_pola_iii.go
package main

import (
  "fmt"
)

func main() {
  var N int
  counter := 0
  fmt.Scan(&N)
  for i := 0; i < N; i++ {
    for j := 0; j < i+1; j++ {
      fmt.Print(counter % 10)
      counter++
    }
    fmt.Println()
  }
}
